CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing guide
+
+To contribute to this project, you'll need [Poetry](https://python-poetry.org/).
+
+```sh
+poetry install --with dev
+
+# run tests
+poetry run pytest
+```
+
+We're also using [tox](https://tox.wiki/en/4.14.2/) to run the tests in all supported environments. Running the full test suite is simple:
```

Comment:
This way, we don't need to update the link all the time ;)

```suggestion
We're also using [tox](https://tox.wiki) to run the tests in all supported environments. Running the full test suite is simple:
```

Could also be set to `https://tox.wiki/en/stable/`.